tests/pytests/functional/transport/ipc/test_pub_server_channel.py

```diff
@@ -93,26 +95,48 @@ def test_issue_36469_tcp(salt_master, salt_minion, transport):
         pytest.skip("Test not applicable to the ZeroMQ transport.")
 
     def _send_small(opts, sid, num=10):
+        loop = asyncio.new_event_loop()
+        asyncio.set_event_loop(loop)
         server_channel = salt.channel.server.PubServerChannel.factory(opts)
-        for idx in range(num):
-            load = {"tgt_type": "glob", "tgt": "*", "jid": "{}-s{}".format(sid, idx)}
-            server_channel.publish(load)
+
+        async def send():
+            await asyncio.sleep(0.3)
+            for idx in range(num):
+                load = {
+                    "tgt_type": "glob",
+                    "tgt": "*",
+                    "jid": "{}-s{}".format(sid, idx),
+                }
+                await server_channel.publish(load)
+
+        asyncio.run(send())
         time.sleep(0.3)
         time.sleep(3)
-        server_channel.close_pub()
+        server_channel.close()
+        loop.close()
+        asyncio.set_event_loop(None)
 
     def _send_large(opts, sid, num=10, size=250000 * 3):
```

Comment:
I know this is existing code, but why 250000, magic number

Reply:
I don't think it's magic, I'ts clear to me that it's just a big number to send a large payload.